aws-glue/jobs/reporting-ratio_stats.py

```diff
@@ -171,7 +171,6 @@ def ccao_mki(fmv, sale_price):
 
 
 def ccao_cod(ratio):
-
```

Comment:
From here down, the rest of the changes in this PR represent formatting tweaks to appease Black/flake8/isort and thehy should not have any effect on code behavior.